docs/standard/linq/toc.yml

```diff
@@ -240,8 +240,8 @@
               href: chain-standard-query-operators-together.md
           - name: "Tutorial: Manipulate content in a WordprocessingML document"
             items:
-            - name: Shape of WordprocessingML documents
-              href: shape-wordprocessingml-documents.md
+            - name: The XML shape of WordprocessingML documents
+              href: xml-shape-wordprocessingml-documents.md
```

Comment:
I see a couple of broken links in this PR for shape-wordprocessingml-documents.md. Can you make sure that existing links to the old file name are fixed?

Reply:
Done - pushed another commit